corehq/motech/tests/test_rate_limiter.py

```diff
@@ -0,0 +1,73 @@
+from unittest import TestCase
+from unittest.mock import patch
+
+from corehq.motech.rate_limiter import _rate_limit_repeater
+from corehq.util.test_utils import flag_enabled
+
+
+@patch('corehq.motech.rate_limiter.metrics_counter')
+@patch('corehq.motech.rate_limiter.repeater_rate_limiter.allow_usage')
+@patch('corehq.motech.rate_limiter.repeater_attempts_rate_limiter.allow_usage')
+@patch('corehq.motech.rate_limiter.global_repeater_rate_limiter.allow_usage')
+class TestRateLimitRepeater(TestCase):
+    @classmethod
+    def setUpClass(cls):
+        cls.domain = 'test-domain'
+        cls.repeater_id = 'test-repeater-id'
+
+    def test_rate_limited(self, global_allowed, repeater_attempts_allowed, repeater_allowed, metrics_counter):
```

Comment:
You're testing 4 booleans:
* global_repeater_rate_limiter
* repeater_rate_limiter
* (feature flag enabled)
* repeater_attempts_rate_limiter

Like @jingcheng16 suggested, maybe state what those booleans are in the test name. e.g.
```python
    def test_global_limited_repeater_limited_attempts_allowed(...):
```